examples/simple/src/posts/PostCreate.tsx

```diff
@@ -141,13 +130,9 @@ const PostCreate = () => {
                 <TextInput autoFocus source="title" />
                 <TextInput source="teaser" fullWidth={true} multiline={true} />
                 <RichTextInput source="body" validate={required()} />
-                <FormSpy subscription={{ values: true }}>
-                    {({ values }) =>
-                        values.title ? (
-                            <NumberInput source="average_note" />
-                        ) : null
-                    }
-                </FormSpy>
+                <DependantField dependency="title">
```

Comment:
Seems like something we could add to core (I'd rename Field To Input)

Reply:
Maybe